Khan/perseus
# Re-enable 'Check Builds' workflow step

## Summary:
The "Check Builds" step compares the size of the builds between HEAD and BASE using 'yarn build:prodsizecheck'.  This command wasn't working before #121.

Issue: None

## Test plan:
- see that the check is successful in GitHub
- create a test PR that shows it can detect deltas in package size, see #124.
